concent_api/utils/tests/test_helpers.py

```diff
@@ -52,3 +57,62 @@ def test_parse_timestamp_to_utc_datetime_should_return_utc_datetime(self):
                 parse_timestamp_to_utc_datetime(timestamp),
                 expected_datetime
             )
+
+
+class TestStorageClusterCertificatePathCheck(TestCase):
+    file_with_wrong_extension = 'certificate.wrong_extension'
+
+    @override_settings(
+            STORAGE_CLUSTER_SSL_CERTIFICATE_PATH = ''
+    )
+    def test_that_empty_string_does_not_produce_any_errors(self):
+        errors = storage_cluster_certificate_path_check()
+
+        assert_that(errors).is_instance_of(list)
+        assert_that(errors).is_empty()
+
+    @override_settings(
+            STORAGE_CLUSTER_SSL_CERTIFICATE_PATH = 'non_existing_path.crt'
+    )
+    def test_that_non_existing_path_produces_an_error(self):
+        expected_error  = Error("File not found")
```

Comment:
Please do not hard-code messages in tests. Checking the error ID instead.

Reply:
@cameel - what's wrong with testing messages?

Reply:
Messages too volatile to test in my opinion. Testing them adds no real benefit and makes some common scenarios harder:
- They can be translated and then they can't really be hard-coded.
- Even if there are no translations (as in this project), hard-coding messages in tests makes it unnecessarily harder to reword them. You then have find and change all instances for something that should be just a simple text change. It's even harder if the message is a template. I commonly need to rewrite badly worded messages and I don't see how hard-coding them in multiple places helps me any way but it sure does make it a lot harder.
    - @rwrzesien proposed using a table of messages and it's a valid solution (we were even going to introduce it at some point), though I think having actual error codes is even better.
- Most importantly, I think it's a bad practice to have code that actually depends on the content of such messages. They're meant to be eventually displayed to the user in the end and be human readable. They're not meant to be parsed, they're not guaranteed not to change between versions and so on. I think that testing them encourages thinking about them like something that can be relied upon. If the code needs to rely on the text it usually means that we're missing machine-readable error codes which can be strictly defined as a part of the API.